defaults/config.js

```diff
@@ -11,7 +11,7 @@ module.exports = {
 	public: true,
 
 	//
-	// Allow connections from this host.
+	// IP address or hostname for the web server to listen on.
```

Comment:
Perhaps specify that listening on 0.0.0.0 will listen on all available interfaces?